kitsune/sumo/static/sumo/scss/layout/_document.scss

```diff
@@ -305,3 +305,54 @@
     }
   }
 }
+
+#document_meta {
+  display: flex;
+  align-items: center;
+  margin-top: 16px;
+  padding-bottom: 16px;
+  margin-bottom: 32px;
+  font-family: 'Inter', sans-serif;
+  font-size: 14px;
```

Comment:
I was thinking that it might be better to replace this with `1rem`, and in general as many of the `px` values with Protocol constants that match.